mender-convert

```diff
@@ -47,9 +62,11 @@ function trap_exit() {
   EXIT_CODE=$?
   if [[ ${EXIT_CODE} -ne 0 && ${EXIT_CODE} -ne ${FATAL_EXIT_CODE} ]]; then
     log_error "mender-convert failed"
-    tac work/convert.log | sed '/DEBUG/q' | tac | sed 's/Running/When running/'
+    [ -e work/convert.log ] && tac work/convert.log | sed '/DEBUG/q' | tac | sed 's/Running/When running/'
     log_error "mender-convert exit code: ${EXIT_CODE}"
   fi
+  [[ -n ${tfile} ]] && rm -f ${tfile}
+  cp work/convert.log convert.log
```

Comment:
Can use `mv` here, should be faster and we are removing the `work` directory on the next line so no need to keep it in there.